.github/workflows/publish-pypi.yml

```diff
@@ -45,6 +44,7 @@ jobs:
         if: github.event_name == 'push'
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
+          # Haven't setup trusted workflow on test pypi yet
```

Comment:
The weird part here is i'm not sure whether we want to use hatch publish vs just the pypi publish workflow which is easy and i've used before.

Reply:
I think it is fine to keep pypa/gh-action-pypi-publish

Reply:
Using hatch to build the wheel and sdist